Source/Core/Simon1994PlanetaryPositions.js

```diff
@@ -80,27 +80,33 @@ define([
             inclination = -inclination;
             longitudeOfNode += CesiumMath.PI;
         }
+        //>>includeStart('debug', pragmas.debug);
```

Comment:
Ah, I don't think there was any.  Just leave it I suppose or add it; I generally separate these pragma blocks with whitespace.